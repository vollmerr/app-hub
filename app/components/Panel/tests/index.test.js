// import React from 'react';
// import { shallow } from 'enzyme';

// import Panel from '../index';

describe('<Panel />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
