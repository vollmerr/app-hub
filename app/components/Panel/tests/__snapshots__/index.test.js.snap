// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Panel /> should render correctly 1`] = `
<Wrapper__Wrapper
  isOpen={true}
>
  <Overlay__Overlay
    onClick={[Function]}
  />
  <Content__Content
    left={false}
  >
    <Children />
  </Content__Content>
</Wrapper__Wrapper>
`;
